### ISSUE TITLE
:nail_care: Drop WC < 6.0 workaround from `full-content` layout

### ISSUE THREAD

**Issue by tomalec** [woocommerce/google-listings-and-ads#1311]
GLA is now compatible with WC 6.3, according to our team tests.

### Describe the issue:

1. The `WC tested up to` was not updated https://github.com/woocommerce/google-listings-and-ads/blob/develop/google-listings-and-ads.php#L15
2. The workaround introduced in https://github.com/woocommerce/google-listings-and-ads/pull/1206/files#diff-1b8407a0e185b837137891f362cd7c3b2e8d553b388a6e4ea73cd11f38de3d45R2-R14 is no longer needed if we also bump `WC requires at least: 5.8` according to L-2

### Steps to reproduce:

1. Open https://github.com/woocommerce/google-listings-and-ads/blob/develop/js/src/css/shared/_woocommerce-admin.scss#L1-L14
2. Check for any no longer needed code :)
3. Thank @eason9487 for making nicely-documented code :pray: 

### Expected behavior:

There amount of legacy code is minimal

### Actual behavior:

there is one rule that most probably is no longer needed

### Additional details:

-  Steps to reproduced the worked around flicker are available at https://github.com/woocommerce/google-listings-and-ads/pull/1206#issuecomment-1017410000

**Comment by mikkamp** [comment]
Just linking a related issue which can be unblocked if we bump the minimum WC version: https://github.com/woocommerce/google-listings-and-ads/issues/1048
